packages/core/components/InputOrGroup/fields/ArrayField/index.tsx

```diff
@@ -1,6 +1,6 @@
 import getClassNameFactory from "../../../../lib/get-class-name-factory";
 import styles from "./styles.module.css";
-import { List, Plus, Trash } from "lucide-react";
+import { List, Plus, Trash, AlertCircle } from "lucide-react";
```

Comment:
I don't think we need the AlertCircle icons here. I think we should just rely on the `disabled` states.